Hymns/Display/DisplayHymnBottomBar.swift

```diff
@@ -4,6 +4,8 @@ import SwiftUI
 struct DisplayHymnBottomBar: View {
 
     @Binding var dialogBuilder: (() -> AnyView)?
+    @Binding var toggleSoundCloud: Bool
```

Comment:
I think we don't need this variable. We already have dialogBuilder to pass information from the bottom bar to the container, so can you just use that instead

Reply:
The problem is I use this variable when the user taps on the soundCloud button. If I use dialogBuilder I am not sure how that would work. So if the user clicks the soundCloud button I pass a view up?

Reply:
Okay, let me play around with this and see if I can get it. I thought of a potential solution where we created a DialogViewModel with customizable properties and pass that in to the Dialog. That way, we have more control over the dialog and we can also introduce a callback for what to do when the dialog is dismissed. I'll try to find time to play with this tomorrow.